src/mgos_imu_mpu6886.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 Google Inc.
```

Comment:
This file should not be copyright Google unless the author works there. Can you check please?

Reply:
removed.
I thought those files inherited the copyright as they are derivatives.